doc/nrf/ug_bootloader_adding.rst

```diff
@@ -104,20 +114,18 @@ Therefore, if a directory in ``/home/user/ncs`` contains signing keys, say ``_ke
 
          -DCONFIG_SB_SIGNING_KEY_FILE=\"/home/user/ncs/_keys/priv.pem\"
 
-      Or, if an environment variable ``NCS`` is set to ``/home/user/ncs``,
+      Or, if you set an environment variable named ``NCS`` to ``/home/user/ncs``,
 
       .. code-block:: console
 
          -DCONFIG_SB_SIGNING_KEY_FILE=\"$NCS/_keys/priv.pem\"
 
-On Unix systems, you can also symbolically link the file into the sample directory, though it will be seen by git as an untracked file.
+On Unix systems, you can also symbolically link the file into the sample directory, even if git will see it as an untracked file.
 
 .. note::
 
-   When using an absolute path, the ``~`` will not be expanded into ``/home/<user>`` in the same way as ``$HOME``.
-   Other environment variables, such as ``$PWD`` or ``$USER``, will also be expanded correctly when used in key file strings.
-
-   In a Kconfig fragment or ``prj.conf`` file, environment variables will not be expanded.
+   The public key string must be an absolute path to the location of the public key file.
+   Environment variables (like ``$HOME``,``$PWD``, or ``$USER``) and the ``~`` character on Unix systems are not expanded when setting an absolute path from a ``prj.conf`` file or Kconfig fragment, but are expanded correctly in key file strings on the command line.
```

Comment:
```suggestion
Environment variables (like ``$HOME``,``$PWD``, or ``$USER``) and the ``~`` character on Unix systems are not expanded when setting an absolute path from a ``prj.conf`` file or Kconfig fragment, but are expanded correctly in key file paths from the command line that are not given as strings.
```

Placing the variables in string arguments to a configuration will not expand them in the command line. For example:

```
west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- -DOVERLAY_CONFIG=\"$PWD/_keys/mykey.pem\"
```
will not expand `$PWD` but this will:
```
west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- -DOVERLAY_CONFIG=$PWD/_keys/mykey.pem
```

Reply:
fixed